projects/public-search/src/app/document-record-search/document-record-search.component.html

```diff
@@ -0,0 +1,27 @@
+<!--
+  RERO ILS UI
+   Copyright (C) 2020 RERO
+  
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3 of the License.
+  
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+  
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<ng-core-record-search [adminMode]="adminMode" [currentType]="currentType" [types]="types" [detailUrl]="detailUrl"
+  [showSearchInput]="showSearchInput" [q]="q" [page]="page" [size]="size" [sort]="sort"
+  (parametersChanged)="updateUrl($event)" (recordsSearched)="recordsSearched($event)"
+>
+  <div class="my-4" *ngIf="currentType === 'documents' && showLink">
+    <a href="#" (click)="linkToGlobalDocuments($event)" translate>
+      Expand to all RERO ILS
```

Comment:
Use the same message than previous : "Expand to all organisations" ? To be more generic

Reply:
The message comes from Nicolas.
done